linux_sql/scripts/psql_docker.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
+#store command in variable
+cmd=$1
+
+if [ $cmd = 'start' ]; then
+	
+	#if password is not provided, ask to enter command again with password
+	if [ "$#" -ne 2 ]; then
+		echo "You have to enter password for database"
+		exit 1
+	fi
+	
+	#store command in variable 
+	pw=$2
```

Comment:
move this to line 5

Reply:
I saved password after checking if the user had entered it or not. Did this so that the script wouldn't require entering password for stopping the container. Should I change that or is it okay?